From 794e180083c9ba1b61a1a63ba40bb729dbad0c0e Mon Sep 17 00:00:00 2001
From: Dmitry Sidorov <dmitry.sidorov@intel.com>
Date: Fri, 31 Mar 2023 13:47:58 +0200
Subject: [PATCH 1/6] Update spirv-as invocations (#1934)

After SPIRV-Tools commit 2e0f4b52 ("tools: refactorize tools flags
parsing. (#5111)", 2023-02-27), spirv-as needs to be told explicitly
when reading from stdin.

Co-authored-by: Sven van Haastregt <sven.vanhaastregt@arm.com>
---
 test/GroupAndSubgroupInstructions.spvasm | 2 +-
 test/spirv-tools-dis.ll                  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/GroupAndSubgroupInstructions.spvasm b/test/GroupAndSubgroupInstructions.spvasm
index d15484b8..9c50a972 100644
--- a/test/GroupAndSubgroupInstructions.spvasm
+++ b/test/GroupAndSubgroupInstructions.spvasm
@@ -31,7 +31,7 @@
 ; spirv-dis tmp.spv -o llvm-spirv/test/GroupAndSubgroupInstructions.spvasm
 
 ; REQUIRES: spirv-as
-; RUN: spirv-as < %s --target-env spv1.0 -o %t.spv
+; RUN: spirv-as %s --target-env spv1.0 -o %t.spv
 ; RUN: spirv-val %t.spv
 ; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv --spirv-target-env=CL1.2 -o %t.bc
 ; RUN: llvm-dis %t.bc -o %t.ll
diff --git a/test/spirv-tools-dis.ll b/test/spirv-tools-dis.ll
index 21409dc3..791f5010 100644
--- a/test/spirv-tools-dis.ll
+++ b/test/spirv-tools-dis.ll
@@ -1,6 +1,6 @@
 ; RUN: llvm-as %s -o %t.bc
 ; RUN: llvm-spirv %t.bc --spirv-tools-dis -o - | FileCheck %s
-; RUN: llvm-spirv %t.bc --spirv-tools-dis -o - | spirv-as
+; RUN: llvm-spirv %t.bc --spirv-tools-dis -o - | spirv-as -
 
 ; Verify that the --spirv-tools-dis options results in SPIRV-Tools compatible assembly.
 
-- 
2.20.1

